### ISSUE TITLE
[Feature Request] Start Earlier Than v12

### ISSUE THREAD

**Issue by opticyclic** [ChrisCarini/jetbrains-ide-release-dates#12]
Start earlier than v12!

Here are a few links to help you out:

https://blog.jetbrains.com/idea/2007/01/intellij-idea-603-is-here/

https://confluence.jetbrains.com/display/IDEADEV/Selena+EAP

https://confluence.jetbrains.com/display/IDEADEV/Diana+EAP

https://confluence.jetbrains.com/display/IDEADEV/Maia+EAP

https://confluence.jetbrains.com/display/IDEADEV/IntelliJ+IDEA+9.0.4+Release+Notes

**Comment by ChrisCarini** [comment]
Hi @opticyclic - thank you for opening this feature request! This project uses the JetBrains API to gather information about the releases for the various JetBrains products and dynamically updates the markdown files with the table of versions.

I think it is best to continue getting the information directly from JetBrains - do you know if there is an existing request to JetBrains to modify their API such that it would include versions older than `11.*`? If not, would you mind creating a feature request for JetBrains to include this information?

**Comment by ChrisCarini** [comment]
@all-contributors please add @opticyclic for content, example, idea

**Comment by opticyclic** [comment]
> Hi @opticyclic - thank you for opening this feature request! This project uses the JetBrains API to gather information about the releases for the various JetBrains products and dynamically updates the markdown files with the table of versions.
> 
> I think it is best to continue getting the information directly from JetBrains - do you know if there is an existing request to JetBrains to modify their API such that it would include versions older than `11.*`? If not, would you mind creating a feature request for JetBrains to include this information?

Would you be open to just add these older ones as a fixed array concatenated with the results from the JetBrains API?

**Comment by ChrisCarini** [comment]
> > Hi @opticyclic - thank you for opening this feature request! This project uses the JetBrains API to gather information about the releases for the various JetBrains products and dynamically updates the markdown files with the table of versions.
> > 
> > I think it is best to continue getting the information directly from JetBrains - do you know if there is an existing request to JetBrains to modify their API such that it would include versions older than `11.*`? If not, would you mind creating a feature request for JetBrains to include this information?
> 
> Would you be open to just concatenating the old versions with the response from the JetBrains API rather than asking them to change theirs?

Yeah, absolutely! It'd be nice to have the information returned from the official API, but it's be fine with it hard-coded here for now. If were able to get enough populated, I'd happily open a request to JetBrains asking them to add to the API.

That being said, I have no concerns with hard-coding for now. Off the top of my head, I'd like to see the following in an implementation before merging it into this repo:

- scalable for all IDEs
- populated with several older versions (just so it doesn't look bare; I'd be ok if it's just for a single IDE to start)
- links in comments to where each release date was found/identified (for posterity)
- (bonus) a section in the readme that provides tips/tricks to find older versions from their wiki/etc (I vaguely recall the being something like this before it became automated from their API; check git history maybe?)

Can you think of anything else?

Are you interested in having a go at the first implementation? I'd be happy to help contribute if you start a feature branch/PR for this request.
